composer.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "wmde/ContentStaging",
+	"description": "",
+	"require-dev": {
+		"squizlabs/php_codesniffer": "~2.1",
+		"phpmd/phpmd": "~2.1"
+	},
+	"scripts": {
+		"cs": [
+			"vendor/bin/phpcs ./*Content* --standard=phpcs.xml --extensions=php -sp",
+			"vendor/bin/phpmd ./*Content* text phpmd.xml"
```

Comment:
This won't work. Try `vendor/bin/phpmd . text phpmd.xml --exclude vendor` or `./vendor/bin/phpmd ContentStaging.i18n.php,ContentStaging.php,SpecialContentStaging.php text phpmd.xml` instead.